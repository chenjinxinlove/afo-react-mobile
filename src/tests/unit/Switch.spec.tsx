import * as React from 'react';
import * as renderer from 'react-test-renderer';
import DemoSwitch from '../../pageages/switch/demo';

describe('Button', () => {
  // 测试快照
  it('test snapshots', () => {
    const component = renderer.create(
      <DemoSwitch />
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
