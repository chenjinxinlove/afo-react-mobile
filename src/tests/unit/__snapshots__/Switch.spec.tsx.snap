// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button test snapshots 1`] = `
<div>
  <p>
    基础用法
  </p>
  <div
    className="afo-switch"
    style={undefined}
  >
    <input
      className="afo-switch__input"
      disabled={undefined}
      onChange={[Function]}
      type="checkbox"
      value={true}
    />
    <i
      className="afo-switch__ui"
    />
    <span
      className="afo-switch__label"
    />
  </div>
  <p>
    禁用状态
  </p>
  <div
    className="afo-switch"
    style={undefined}
  >
    <input
      className="afo-switch__input"
      disabled={true}
      onChange={[Function]}
      type="checkbox"
      value={true}
    />
    <i
      className="afo-switch__ui"
    />
    <span
      className="afo-switch__label"
    />
  </div>
</div>
`;
